loglens

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+import json
+import sys
+
+for line in sys.stdin:
+    record = json.loads(line)
+
+    # Output the record as Pretty-Print JSON
+    # print(json.dumps(record, indent=4, separators=(', ', ': ')))
+
+    # Output the record like JADE does
+    # %d [%t] %-5p (%C:%L) - %m%n
+    # 2019-01-15 00:03:48,053 [ConcurrentDownload-0] INFO  (edu.wisc.icecube.jade.process.downloader.ConcurrentDownloader:108) - /mnt/data/jade/download_pending/s333_190113_0020_8d2b5d30-060a-4b7c-9153-162e2c15d417 size 910499840 matches SPTR s333_190113_0020_8d2b5d30-060a-4b7c-9153-162e2c15d417 size 910499840
+    timestamp = record["timestamp"].replace("T", " ").replace(".", ",")[0:23]
+    threadName = record["threadName"]
```

Comment:
I really don't think the thread name matters to us, since there is only ever one thread.

Reply:
I'll keep it for now, because when I piped `picker.sh` into `loglens`, I got this:
```
2019-01-17 16:11:08,962 [MainThread] INFO  (picker.py:130) - Starting status loop
2019-01-17 16:11:08,962 [MainThread] INFO  (picker.py:99) - Sending status heartbeat
2019-01-17 16:11:08,963 [MainThread] INFO  (picker.py:111) - PATCH http://127.0.0.1:9090/status/picker - {'bluebird-picker': {'timestamp': '2019-01-17T16:11:08.963053', 'file_catalog_ok': False, 'last_work_begin_timestamp': '2019-01-17T16:11:08.961677', 'last_work_end_timestamp': '2019-01-17T16:11:08.961677', 'lta_ok': False}}
2019-01-17 16:11:08,963 [MainThread] INFO  (picker.py:140) - Starting work loop
2019-01-17 16:11:08,963 [MainThread] INFO  (picker.py:74) - Starting picker work cycle
2019-01-17 16:11:08,964 [MainThread] INFO  (picker.py:92) - Ending picker work cycle
2019-01-17 16:11:08,966 [ThreadPoolExecutor-0_0] DEBUG (connectionpool.py:205) - Starting new HTTP connection (1): 127.0.0.1:9090
2019-01-17 16:11:08,967 [MainThread] ERROR (picker.py:122) - ConnectionError trying to PATCH /status/picker with heartbeat
```